linux_os/guide/system/software/integrity/crypto/harden_ssh_client_crypto_policy/rule.yml

```diff
@@ -5,11 +5,13 @@ title: 'Harden SSH client Crypto Policy'
 description: |-
     Crypto Policies are means of enforcing certain cryptographic settings for selected applications including OpenSSH server.
     The SSH client is by default configured to modify its configuration based on currently configured Crypto-Policy. However, in certain cases it might be needed to override the Crypto Policy specific to OpenSSH Client and leave rest of the Crypto Policy intact.
-    This is unfortunately not possible through the Crypto Policy mechanism directly due to the way in which the SSH Client parses configuration files. It is needed to apply Crypto Policy modifications by dropping a file into the <tt>/etc/ssh/ssh_config.d/</tt> directory. The file needs to be named in a way such that it is parsed by SSH client before the actual Crypto Policy.
+    This is unfortunately not possible through the Crypto Policy mechanism directly due to the way in which the SSH Client parses configuration files. During the parsing process, as soon as the client parses some configuration option and its value, it remembers it and ignores any subsequent overides. The mechanism of crypto policies appends eventual customizations at the end of the system wide crypto policy. Therefore customizations will be ignored by the client.
```

Comment:
Suggestion: `not possible through the Crypto Policy mechanism directly due to ...` to `not directly possible through the Crypto Policy mechanism due to...` 
Typo: `overrides`.

Reply:
`The mechanism of crypto policies appends eventual customizations at the end of the system wide crypto policy. Therefore customizations will be ignored by the client.`

Not sure what this means. crypto-policy mechanism append customizations at the end of crypto-policy, and because of this, they are ignored by the client?

Maybe you want to say that "due to how SSH Client reads the crypto-poicy, any setting that a "official" crypto-policy would configure, will be ignored by the client" ?

Reply:
I want to say that if you go the /etc/crypto-policies/local.d way, in the case of Openssh client this approach will not work if you try to override a parameter which is already configured by the system wide policy.

Reply:
Aha! I understand this as part of `This file subsequently includes the actual system-wide crypto policy placed in <tt>/etc/crypto-policies/back-ends/openssh.config</tt>.`
As any file in ` /etc/crypto-policies/local.d` will become part of what is in `back-ends/openssh.config`.

To me this seems a bit too deep in details for the rule description.